app/assets/stylesheets/main.scss

```diff
@@ -53,3 +57,49 @@
 .main-button:hover {
   color: white;
 }
+
+.small-button {
+  text-decoration: none;
+  font-weight: bold;
+  font-size: 18px;
+  padding: 10px 25px;
+  border: 2px solid black;
+  background-color: purple;
+  color: white;
+  border-radius: 6px;
+}
+.small-button:hover {
+  color: white;
+}
+
+.user-entry{
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  border: 1px solid #ff96ff;
+  padding: 15px 20px;
+}
+
+.nav-bar{
+  display: flex;
+  flex-direction: column;
+  padding: 0;
+}
+.nav-button{
```

Comment:
@umair-khalid-brainx 
Kindly try to format code before committing code.
You can use shortcut to format it.

In above code there should be space before `{`